src/test/java/com/google/cloud/genomics/dataflow/pipelines/CountReadsITCase.java

```diff
@@ -107,6 +108,36 @@ public void testCloud() throws Exception {
   }
 
   /**
+   * CountReads running on Dataflow with API input.
+   */
+  @Test
+  public void testCloudWithAPI() throws Exception {
+    final String OUTPUT = TEST_OUTPUT_GCS_FOLDER + "CountReadsITCase-testCloudWithAPI-output.txt";
+    String[] ARGS = {
+        "--apiKey=" + API_KEY,
+        "--project=" + TEST_PROJECT,
+        "--output=" + OUTPUT,
+        "--numWorkers=2",
+        "--runner=BlockingDataflowPipelineRunner",
+        "--stagingLocation=" + TEST_STAGING_GCS_FOLDER,
+        "--references=1:550000:560000",
+        "--readGroupSetId=" + TEST_READGROUPSET
+    };
+    final long EXPECTED = 685;
```

Comment:
This looks good!

Consider moving this constant (and the one on line 96) up with the GCS path and the ReadGroupSetId constants since they all go together and match.